Makefile

```diff
@@ -46,20 +45,21 @@ test:
 	@$(MOCHA) \
 		--timeout 120s \
 		--slow 3s \
-		--grep "$(FILTER)" \
```

Comment:
In theory `make tests` is used to run all tests. And if you wanna filter some tests you would like to use test-watch. At least I thought that. Anyway ... no make sense remove --grep here so I've restored this parameter.